Commander X16 Programmer's Reference Guide.md

```diff
@@ -472,6 +474,17 @@ The string can contain printable ASCII characters (`CHR$($20)` to `CHR$($7E)`),
 	20 FORI=1TO20:RECTRND(1)*320,RND(1)*200,RND(1)*320,RND(1)*200,RND(1)*256:NEXT
 	30 GOTO20
 
+#### RESET
+
+**TYPE: Command**
+**FORMAT: RESET**
+
+**Action:** Performs a reset.
```

Comment:
Someone might think that it resets only the BASIC interpreter.  Therefore, I suggest saying,

Performs a software reset of the computer system.

Reply:
Sounds good, thanks for the review

Reply:
done